src/main/java/com/google/api/gax/grpc/PageStreamingCallSettings.java

```diff
@@ -58,10 +58,8 @@
     return createBaseCallable(channel, executor);
   }
 
-  /**
-   * Package-private, for use by ApiCallable.
-   */
-  ApiCallable<RequestT, PageAccessor<ResourceT>> createPagedVariant(
+  /** Package-private, for use by ApiCallable. */
+  ApiCallable<RequestT, PagedListResponse<RequestT, ResponseT, ResourceT>> createPagedVariant(
```

Comment:
I know, I didn't like this very much either as I was doing it. I don't know if there is any alternative though